### DISCUSSION TITLE
Exact meaning of user_count (API)?

### DISCUSSION THREAD

**Discussion by Der-K-2000** [mastodon/mastodon#21146] [General]
In [/api/v1/instance/activity](https://docs.joinmastodon.org/methods/instance/) there are some interesting fields, but I'm unsure about `user_count`.

Does this number also include 

1. the users who moved to other instances (but have still an abandoned account) and
2. are there also accounts which are suspended?

Maybe (yes, other topic) the api-endpoints to this could be a bit more enhanced.

**Comment by trwnh** [comment]
It's just total accounts, I think -- it would include moved accounts, and it would include suspended accounts that haven't been deleted yet (but only until they're deleted)

**Comment by Der-K-2000** [comment]
Thanks for your estimation! I suppose the same, but I'd like to have it verified.

**Comment by mjankowski** [comment]
It's the count of all confirmed non-suspended users.
